includes/class-stats.php

```diff
@@ -2096,7 +2096,7 @@ public function get_customer_order_count( $query = array() ) {
 		$this->pre_query( $query );
 
 		// Only `COUNT` and `AVG` are accepted by this method.
-		$accepted_functions = array( 'COUNT', 'AVG' );
+		$accepted_functions = array( 'COUNT', 'AVG(id)' );
 
 		$function = isset( $this->query_vars['function'] ) && in_array( strtoupper( $this->query_vars['function'] ), $accepted_functions, true )
 			? $this->query_vars['function'] . "({$this->query_vars['column']})"
```

Comment:
@LisaCee

Now, with your change the [existing stats usage](https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/includes/admin/reporting/reports.php#L2373) will still not use the correct query:

```php
$stats = new EDD\Stats();
$stats->get_customer_order_count( array(
  'function' => 'AVG',
) );
```

This does not pass the `in_array()` check above now (`AVG` is not in `COUNT` or `AVG(id)`).

Instead I think we could keep the array of accepted functions the same and if the passed function string passes validation set that as the value, falling back to an empty string. If the passed function is not considered valid/accepted and we end up with an empty string we can just return early with a `0` value because we don't have a way of determining which SQL query to run.

The original implementation of this stats method half written to be flexible (referencing dynamic column values in `$this->query_vars['column']` and dynamic function values in `$this->query_vars['function']`) with the other half uses very static comparisons and SQL statements. 

Because we know the SQL calculations for determining the average or total counts can really only happen one way I think it makes sense to remove some of the "dynamicness" if this method and hard code the calculation portions of the SQL queries (which is already happening for `AVG`) and simplify the checks for the type of count we want.